hack/update-openapigen.sh

```diff
@@ -5,11 +5,13 @@ set -o nounset
 set -o pipefail
 
 PROJECT_ROOT=$(cd $(dirname "$0")/.. ; pwd)
-CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
+# CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
+OPENAPIGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/kube-openapi
 VERSION="v1alpha1"
 
 # Sensor
```

Comment:
You may want to remove the comment #Sensor

Reply:
yep. Still in WIP. adding a few other things

Reply:
yes. removed that comment.